examples/configs/indent_anchor.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative '../helper'
+
+current_offset_config       = Oppen::Config::IndentAnchor::CURRENT_OFFSET
+end_of_previous_line_config = Oppen::Config::IndentAnchor::END_OF_PREVIOUS_LINE
+printer_with_config = Oppen::Wadler.new config: Oppen::Config.new(indent_anchor: current_offset_config), width: 13
+printer_no_config   = Oppen::Wadler.new config: Oppen::Config.new(indent_anchor: end_of_previous_line_config), width: 13
```

Comment:
no need to redefine var, you may want to do something like this:

```suggestion
printer_with_config = 
  Oppen::Wadler.new(
    config: Oppen::Config.new(indent_anchor: Oppen::Config::IndentAnchor::CURRENT_OFFSET), 
    width: 13,
  )
printer_no_config = 
  Oppen::Wadler.new(
    config: Oppen::Config.new(indent_anchor: Oppen::Config::IndentAnchor::END_OF_PREVIOUS_LINE), 
    width: 13
  )
```

Reply:
I just got an idea:; what if:

1. `Oppen::Config::IndentAnchor::END_OF_PREVIOUS_LINE` and all other "enums" we defines, were actually sybols, i.e. `:end_of_previous_line`
2. We provide easy access to those through the module `Oppen`, directly, by defining something like:

```ruby
module Oppen
  def self.end_of_previous_line = Config::IndentAnchor::END_OF_PREVIOUS_LINE
  # etc …
end
```

or

```ruby
module Oppen
  module Config
    def self.end_of_previous_line = Config::IndentAnchor::END_OF_PREVIOUS_LINE
    # etc …
  end
end
```

There shouldn't be any name clashes between all the `enums` we define.

If so, we need to document them using Yard's `@group`. Let's meet and discuss this.